### DISCUSSION TITLE
paid content ??

### DISCUSSION THREAD

**Discussion by crowwx** [UltimaHoarder/UltimaScraper#712] [Q&A]
Hi,
So everything wokrs fine, but it can't load/find any of the paid content of all the onlyfans accounts i choosed, the paid folder is always empty, unfortunately there are many paid content to see.
Any solution please? THANKS.

**Comment by James-MF-Thomas** [comment]
he's looking for unpaid  pay content.. I'll upvote this, so this gets resolved for you as quickly as possible. we need to get that feature working for you
